### PR TITLE
docs: Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by zyxiaoyuer**

### CODE REVIEW

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 2.10.0 - Nov 15, 2021 
+
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+New Features:
+- Enable developers with the capability to extend Teams apps to run across Microsoft 365, get instructions from our [documentation](https://aka.ms/teamsfx-extend-m365). 
+- Provide Teams sample apps which can run across Microsoft 365. Users can get a initial experience of running Teams apps in Microsoft 365.
```
Comment (kimizhu) [inline packages/vscode-extension/CHANGELOG.md:9]: an initial

Comment (kimizhu) [inline packages/vscode-extension/CHANGELOG.md:9]: Should we give the name of the samples here?

Comment (zyxiaoyuer) [reply packages/vscode-extension/CHANGELOG.md]: corrected

Comment (zyxiaoyuer) [reply packages/vscode-extension/CHANGELOG.md]: added

File: packages/vscode-extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+
+Enhancement:
+
+- Improve the create new project experience. Remove the selection of Frontend Hosting Type.
```
Comment (kimizhu) [inline packages/vscode-extension/CHANGELOG.md:13]: This sprint is dedicated to MetaOS, so I don't think this enhancement will be shipped,

**Review by kimizhu — APPROVED** [review]

**Review by wenytang-ms — APPROVED** [review]

**Pull request merged**